test/index.js

```diff
@@ -44,13 +48,14 @@ function triggerClick(el, modified){
             window, null,
             0, 0, 0, 0, /* coordinates */
             !!modified, false, false, false, /* modifier keys */
-            0 /*left*/,
+            button,
             null
         );
         el.dispatchEvent(ev);
     } else if (document.createEventObject) {
         ev = document.createEventObject();
         ev.ctrlKey = !!modified;
+        ev.button = button;
```

Comment:
maybe this doesn't do what I expect it to in IE? I don't have an IE to test in at the moment..